pipenv/cli/command.py

```diff
@@ -760,24 +760,24 @@ def requirements(state, dev=False, dev_only=False, hash=False):
     for i, package_index in enumerate(lockfile["_meta"]["sources"]):
         prefix = "-i" if i == 0 else "--extra-index-url"
         echo(" ".join([prefix, package_index["url"]]))
+
+    from pipenv.utils.dependencies import convert_deps_to_pip
+
+    lockfile = state.project.lockfile_content
+    deps = {}
+
     if not dev_only:
-        for req_name, value in lockfile["default"].items():
-            if value.get("editable", False):
-                echo("-e " + value["path"])
-            elif hash:
-                hashes = [f" \\\n    --hash={h}" for h in value.get("hashes", [])]
-                echo("".join([req_name, value["version"], *hashes]))
-            else:
-                echo("".join([req_name, value["version"]]))
+        deps.update(lockfile["default"])
     if dev or dev_only:
-        for req_name, value in lockfile["develop"].items():
-            if value.get("editable", False):
-                echo("-e " + value["path"])
```

Comment:
Does `convert_deps_to_pip` handle editable dependencies?

Reply:
Yes, it does.